pyapns_client/notification.py

```diff
@@ -178,8 +178,16 @@ def to_dict(self, alert_body: typing.Union[str, None] = None):
             dict: A dictionary representation of the payload.
         """
         d = {"aps": {}}
-        if self.alert is not None:
-            d["aps"]["alert"] = self.alert.to_dict(alert_body=alert_body)
+        if self.alert:
+            if isinstance(self.alert, _PayloadAlert):
```

Comment:
Are these checks still necessary? Since `self.alert` can only be an instance of `_PayloadAlert` or `None`, it may not be required to perform these checks. And we can use last version of code: `d["aps"]["alert"] = self.alert.to_dict(alert_body=alert_body)`

Reply:
:100: No longer needed now everything is centralized and alerts are automatically converted to dictionaries. Should be fixed now. :smile: